setup.py

```diff
@@ -39,6 +42,7 @@
 DEVELOPMENT_REQUIREMENTS = [
     "pytest<8.0.0",  # check SNOW-1022240 for more details on the pin here
     "pytest-cov",
```

Comment:
do you recall where we use/why we need "wrapt"?

Reply:
Looking into it now.

Reply:
`wrapt` appears unused. I dropped it. We'll see what breaks. We can reintroduce it when necessary.